Buildings/Examples/VAVReheat/BaseClasses/VAVReheatBox.mo

```diff
@@ -0,0 +1,276 @@
+          lineColor={0,0,0}),
+        Polygon(
+          points={{-20,-20},{14,-20},{14,-22},{-20,-22},{-20,-20}},
+          pattern=LinePattern.None,
+          smooth=Smooth.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          lineColor={0,0,0}),
+        Polygon(
+          points={{-20,-44},{14,-44},{14,-46},{-20,-46},{-20,-44}},
+          pattern=LinePattern.None,
+          smooth=Smooth.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          lineColor={0,0,0}),
+        Polygon(
+          points={{0,-26},{14,-20},{14,-22},{0,-28},{0,-26}},
+          pattern=LinePattern.None,
+          smooth=Smooth.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          lineColor={0,0,0}),
+        Polygon(
+          points={{0,-26},{14,-32},{14,-34},{0,-28},{0,-26}},
+          pattern=LinePattern.None,
+          smooth=Smooth.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          lineColor={0,0,0}),
+        Polygon(
+          points={{0,-38},{14,-44},{14,-46},{0,-40},{0,-38}},
+          pattern=LinePattern.None,
+          smooth=Smooth.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          lineColor={0,0,0}),
+        Polygon(
+          points={{0,-38},{14,-32},{14,-34},{0,-40},{0,-38}},
+          pattern=LinePattern.None,
+          smooth=Smooth.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          lineColor={0,0,0}),
+        Rectangle(
+          extent={{-100,-18},{-20,-24}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-100,-42},{-20,-48}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-12,3},{12,-3}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Solid,
+          origin={-97,-12},
+          rotation=90),
+        Rectangle(
+          extent={{-12,3},{12,-3}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Solid,
+          origin={-97,-54},
+          rotation=90)}),       Documentation(info="<html>
+<p>
+Model for a VAV terminal box with a water reheat coil and pressure independent damper.
+The pressure independent damper model includes an idealized flow rate controller
+and requires a discharge air flow rate set-point (normalized to the nominal value)
+as a control signal.
+</p>
+</html>", revisions="<html>
+<ul>
+<li>
+February 12, 2021, by Baptiste Ravache:<br/>
+First implementation, based on <code>Buildings.Obsolete.Examples.VAVReheat.BaseClasses.VAVBranch</code><br/>
```

Comment:
Use a hyperlink, e.g.,
```
<a href=\"modelica://Buildings.Obsolete.Examples.VAVReheat.BaseClasses.VAVBranch\">
Buildings.Obsolete.Examples.VAVReheat.BaseClasses.VAVBranch</a>
```
This way, the correct name is checked when exporting the html documentation, and users can easier navigate to it.